docs/source/delta-clustering.md

```diff
@@ -87,6 +97,16 @@ OPTIMIZE table_name;
 
 Liquid clustering is incremental, meaning that data is only rewritten as necessary to accommodate data that needs to be clustered. Already clustered data files with different clustering columns are not rewritten.
 
+In <Delta> 3.3 and above, you can force reclustering of all records in a table with the following syntax:
+
+```sql
+OPTIMIZE table_name FULL;
+```
+
+.. important:: Running `OPTIMIZE FULL` reclusters all existing data as necessary. For large tables that have not previously been clustered on the specified keys, this operation might take hours.
+
+Run `OPTIMIZE FULL` when you enable clustering for the first time or change clustering keys. If you have previously run `OPTIMIZE FULL` and there has been no change to clustering keys, `OPTIMIZE FULL` runs the same as `OPTIMIZE`. Always use `OPTIMIZE FULL` to ensure that data layout reflects the current clustering keys.
```

Comment:
`Run `OPTIMIZE FULL` when you enable clustering for the first time` is this needed?

Reply:
nope, removed.